src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -2391,6 +2403,22 @@ public static void provideAndGenerateSampleElements(Random random, Workspace wor
 				addGeneratableElementAndAssert(workspace, lootTable);
 			}
 		}
+
+		// add sample banner (used by test mod elements) if supported
+		if (workspace.getGeneratorStats().getModElementTypeCoverageInfo().get(ModElementType.BANNERPATTERN)

```

Comment:
Since you specify banner patter order priority, sample patterns will be generated before item ME, so you should be able to specify banner patterns by the ME name of test patterns there directly as "ExamplebannerpatternN"